src/Lib/Elebee.php

```diff
@@ -358,27 +291,33 @@ private function defineAdminHooks() {
      *
      */
     private function definePublicHooks() {
+        if ( is_admin() && !is_customize_preview() ) {
+            return;
+        }
 
         $this->loader->addAction( 'init', Config::class, 'cleanUpHead' );
-        $this->loader->addAction( 'init', Config::class, 'disableEmojies' );
         $this->loader->addFilter( 'status_header', Config::class, 'disableRedirectGuess' );
 
         $elebeePublic = new ElebeePublic( $this->getThemeName(), $this->getVersion() );
 
         $this->loader->addAction( 'wp_head', $elebeePublic, 'embedGoogleAnalytics', 0 );
         $this->loader->addAction( 'wp_head', $elebeePublic, 'embedIEConditionals', 0 );
-        $this->loader->addAction( 'wp_enqueue_styles', $elebeePublic, 'enqueueStyles' );
+        $this->loader->addAction( 'wp_enqueue_scripts', $elebeePublic, 'enqueueStyles' );
```

Comment:
okay we are enqueueStyles as wp_enqueue_scripts?

Reply:
Yes, because there is no hook explicitly for styles.